test/test_force.js

```diff
@@ -237,8 +237,10 @@ function createCompileApp(tmpDir, os, actualAppType, templateRepoUri, pluginRepo
     var isHybrid = actualAppType.indexOf('hybrid') == 0;
     var isHybridRemote = actualAppType === APP_TYPE.hybrid_remote;
     var target = actualAppType + ' app for ' + os + (templateRepoUri ? ' based on template ' + getTemplateNameFromUri(templateRepoUri) : '');
-    var appName = actualAppType + os + 'App';
-    var packageName = (actualAppType == 'native') ? 'com.salesforce.' + actualAppType + '_java' : 'com.salesforce.' + actualAppType;
```

Comment:
Why are we appending `_java` for `Android`? It could be a `Kotlin` app. Can we do `com.salesforce.hybrid` and `com.salesforce.native` on both platforms?

Reply:
@bhariharan App type for Kotlin is already `native_kotlin`.  Package names should be unique for different apps, right?  Also `packageName` is how the UI tests know what app (an unique elements in that app) to assert for.